internal/handlers/metrics.go

```diff
@@ -1,43 +0,0 @@
-package handlers
```

Comment:
This is technically a breaking change… Although I wouldn't expect anyone to use it apart from the old dashboard, maybe we should consider having a separate "internal" api for stuff that's really only relevant for our UI where we can easily make changes like this. Can be considered for future features I guess…